src/resample.c

```diff
@@ -71,17 +62,19 @@ main (int argc, char *argv [])
 	for (k = 1 ; k < argc - 2 ; k++)
 	{	if (strcmp (argv [k], "--max-speed") == 0)
 			max_speed = SF_TRUE ;
+		else if (strcmp (argv [k], "--no-normalize") == 0)
+			normalize = 0 ;
 		else if (strcmp (argv [k], "-to") == 0)
 		{	k ++ ;
-			new_sample_rate = parse_int_or_die (argv [k], "sample rate") ;
+			new_sample_rate = atoi (argv [k]) ;
 			}
```

Comment:
The `parse_int_or_die` function has *very* different semantics to `atoi`.

I thin the semantics is important and I don't mind if you copy over the `parse_int_or_die` function.